.changeset/fresh-gifts-smile.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-techdocs-module-addons-contrib': patch
+---
+
+textsize-fix value label text color
```

Comment:
```suggestion
Fixed the value label text color in dark mode for the TextSize addon.
```

Reply:
I don't have access to commit this update to the changeset. If you could update the changeset to be more descriptive of what was fixed that would be great. Happy to merge after!

Reply:
thanks @squid-ney ! adjustments made